<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentManagement</class>
 <widget class="QMainWindow" name="StudentManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#StudentManagement{
	background-color: rgb(255,255,255);
}

#Header {
	background-color: rgb(165, 213, 255);
}

#Header #Logo{
	image: url(:/Pic/logo.png);
	border: none;
}

#Header #NameSW{
	font-family: &quot;Robotol&quot;, sans-serif;
	font-size: 25px;
	font-weight: bold;
	text-algin: left;
}

#result_frame {
	border-radius: 10px;
	border: 1px solid black;
	background-color: #fff;
}

QTableWidget {
	border-radius: 3px;
	border: 1px solid #f0f0f0;
}

QHeaderView::section {
	border: none;
	border-bottom: 1px solid black;
	text-align: left;
	padding: 3px 5px;
}

QTableWidget::Item {
	border-bottom: 1px solid rgb(212, 212, 212);
	color: #000;
	padding-left: 3px;
}

#btn_frame {
	border: 1px solid black;
	border-radius: 10px;
	background-color: rgb(255,255,255);
}

#btn_frame QPushButton{
	background-color: #a5d5ff;
	border-radius: 10px;
}

#btn_frame #delete_btn{
	background-color: rgb(255, 111, 111);
}

#btn_frame #export_btn{
	background-color: rgb(75, 225, 0);
}

#btn_frame QPushButton:hover{
	border-color: rgb(65, 173, 255);
}





</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="Header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="NameSW">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Robotol</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>STUDENT INFORMATION MANAGEMENT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_class">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Robotol</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1281</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="result_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>1211</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>StudentID</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Email</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Faculty</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Major</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Year</string>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mark</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="btn_frame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>1211</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="update_btn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delete_btn">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="read_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Diem_btn">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Mark</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_btn">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QPushButton" name="class_btn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Select Class</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
